doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1076 @@
+* Support for the Zigbee Cluster Library specification v8 (ZCL 8) and Base Device Behavior specification v3.0.1 (BDB 3.0.1) is no longer experimental.
+  ZCL 8 and BDB 3.0.1 libraries are now used by default in Zigbee samples.
+* Updated Zephyr LwM2M stack to v1.1, which provides extended commands and options for more efficient data transmission.
+* Added nRF Cloud Location Services support in the AVSystem's Coiote LwM2M server.
+* Added power consumption optimization for poor satellite coverage when using GNSS.
+
+A :ref:`migration guide <ncs_2.0.0_migration>` is available for users moving from |NCS| v1.x to v2.x.
+
+Sign up for the `nRF Connect SDK v2.0.0 webinar`_ to learn more about the new features.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.0.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.0.0`_.
+
+IDE support
+***********
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.0.0.
+SEGGER Embedded Studio Nordic Edition is no longer tested and recommended for new projects.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.0.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.0.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.0.0`_ for the list of issues valid for the latest release.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Application development
+=======================
+
+* Fixed an issue with |NCS| Toolchain delimiter handling on MacOS, which could in special situations result in the build system not being able to properly locate the correct program needed.
+
+Boards
+======
+
+* Added necessary reset functionality for using the nRF52840 SoC on the Thingy:91 as a Bluetooth controller.
+
```

Comment:
```suggestion
Application development
=======================

This section provides lists of changes related to :ref:`ug_app_dev`.

Board support
-------------

* Added necessary reset functionality for using the nRF52840 SoC on the Thingy:91 as a Bluetooth controller.

Build system
-------------

* Fixed an issue with |NCS| Toolchain delimiter handling on MacOS, which could in special situations result in the build system not being able to properly locate the correct program needed.

```
Technically speaking, we're mirroring the doc hierarchy from the menu on the left in the release notes, so I'm trying to do this with this edit, just for consistency.